scripts/upgrade.sh

```diff
@@ -64,6 +65,7 @@ echo "Host is $node_host"
 echo "Node is: " $node_role
 printf "BootKey is "; mask $boot_key
 printf "HarvestKey is "; mask $harvest_key
+printf "ReplicatorKey is "; mask $replicator_key
```

Comment:
I think we need a new util or script for first time users to onboard the replicatorkey.